stack/values.yaml

```diff
@@ -1,7 +1,5 @@
 timescaledb-single:
   enabled: true
-  loadBalancer:
```

Comment:
I had to do this because I was getting the errror:

```
Error: values don't meet the specifications of the schema(s) in the following chart(s):
timescaledb-single:
- (root): Additional property loadBalancer is not allowed

make[1]: *** [install] Error 1
make: *** [stack] Error 2
```

Reply:
That is due to this https://github.com/timescale/helm-charts/commit/44c225c0a6cce1904bd9b0214a76776f3439198c